src/index.html

```diff
@@ -1,14 +1,17 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body class="container">
+    <div class="watch">
+      <div class="watch__small-hand"></div>
+      <div class="watch__big-hand"></div>
+    </div>
+  </body>
```

Comment:
The only change I can suggest is that:  Checklist p.1 - Don't forget about semantics, div development is not a better option; I don't think it is really matters in such small project but checklist is a checklist, so it's better to change that
<sub>Anyway I'll approve that 'cause I don't consider this as a really big mistake or something</sub>